main.py

```diff
@@ -2,6 +2,67 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
+
+def crud_for_genres() -> QuerySet:
+    # Create a new genres in table:
+    genres = ["Western", "Action", "Dramma"]
+    for genre_name in genres:
+        Genre.objects.create(name=genre_name)
+
+    # Update an existing genre name to a new:
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+
+    # Deleting an existing genre from table:
+    Genre.objects.filter(name="Action").delete()
+
+    # Read data from table:
+    return Genre.objects.all()
+
+
+def crud_for_actors() -> QuerySet:
+    # Create new actors inputs in table:
+    actors = [
+        {
+            "first_name": "George", "last_name": "Klooney"
```

Comment:
There is a typo in the actor's last name 'Klooney'. It should be 'Clooney'. This is corrected later in the update operation, but it's better to fix it here as well to maintain consistency.